### ISSUE TITLE
Unable to resolve certain dependency during project creation

### ISSUE THREAD

**Issue by 0xstragner** [ton-org/blueprint#64]
1. Run interactive project builder
```shell
npm create ton@latest 
```

2. Provide name & first contract name
3. Error occurred

## Output:
```
[1/3] Copying files...
[2/3] Installing dependencies...

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: hallo-contracts@0.0.1
npm ERR! Found: ton-core@0.49.1
npm ERR! node_modules/ton-core
npm ERR!   dev ton-core@"^0.49.0" from the root project
npm ERR!   peer ton-core@">=0.48.0" from @ton-community/blueprint@0.12.0
npm ERR!   node_modules/@ton-community/blueprint
npm ERR!     dev @ton-community/blueprint@"^0.12.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ton-core@">=0.51.0" from ton@13.6.0
npm ERR! node_modules/ton
npm ERR!   dev ton@"^13.4.1" from the root project
npm ERR!   peer ton@">=13.4.1" from @ton-community/blueprint@0.12.0
npm ERR!   node_modules/@ton-community/blueprint
npm ERR!     dev @ton-community/blueprint@"^0.12.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-debug-0.log
Error: Command failed: npm install
    at checkExecSyncError (node:child_process:885:11)
    at execSync (node:child_process:957:15)
    at main (/Users/stragner/.npm/_npx/b56225140edd990c/node_modules/create-ton/dist/cli.js:97:42) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 19504,
  stdout: null,
  stderr: null
}
```

**Comment by 0xstragner** [comment]
```
0 verbose cli /opt/homebrew/Cellar/node/20.2.0/bin/node /opt/homebrew/bin/npm
1 info using npm@9.8.1
2 info using node@v20.2.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 0ms
5 timing config:load:file:/opt/homebrew/lib/node_modules/npm/npmrc Completed in 1ms
6 timing config:load:builtin Completed in 2ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/Users/stragner/Developer/lumoscompany/hallo-contracts/hallo-contracts/.npmrc Completed in 1ms
10 timing config:load:project Completed in 2ms
11 timing config:load:file:/Users/stragner/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/opt/homebrew/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 0ms
16 timing config:load Completed in 5ms
17 timing npm:load:configload Completed in 5ms
18 timing config:load:flatten Completed in 1ms
19 timing npm:load:mkdirpcache Completed in 0ms
20 timing npm:load:mkdirplogs Completed in 0ms
21 verbose title npm install
22 verbose argv "install"
23 timing npm:load:setTitle Completed in 2ms
24 timing npm:load:display Completed in 0ms
25 verbose logfile logs-max:10 dir:/Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-
26 verbose logfile /Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-debug-0.log
27 timing npm:load:logFile Completed in 4ms
28 timing npm:load:timers Completed in 0ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 20ms
31 timing arborist:ctor Completed in 0ms
32 silly logfile start cleaning logs, removing 1 files
33 timing arborist:ctor Completed in 0ms
34 silly logfile done cleaning log files
35 timing idealTree:init Completed in 6ms
36 timing idealTree:userRequests Completed in 0ms
37 silly idealTree buildDeps
38 silly fetch manifest @ton-community/blueprint@^0.12.0
39 http fetch GET 200 https://registry.npmjs.org/@ton-community%2fblueprint 6ms (cache hit)
40 silly fetch manifest ton@^13.4.1
41 http fetch GET 200 https://registry.npmjs.org/ton 2ms (cache hit)
42 silly fetch manifest ton-core@^0.49.0
43 http fetch GET 200 https://registry.npmjs.org/ton-core 1ms (cache hit)
44 silly fetch manifest ton-core@>=0.51.0
45 silly fetch manifest ton-crypto@^3.2.0
46 http fetch GET 200 https://registry.npmjs.org/ton-crypto 1ms (cache hit)
47 timing idealTree Completed in 27ms
48 timing command:install Completed in 32ms
49 verbose stack Error: unable to resolve dependency tree
49 verbose stack     at #failPeerConflict (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1376:25)
49 verbose stack     at #loadPeerSet (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1342:33)
49 verbose stack     at async #loadPeerSet (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1321:23)
49 verbose stack     at async #buildDepStep (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:917:11)
49 verbose stack     at async Arborist.buildIdealTree (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:196:7)
49 verbose stack     at async Promise.all (index 1)
49 verbose stack     at async Arborist.reify (/opt/homebrew/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/reify.js:159:5)
49 verbose stack     at async Install.exec (/opt/homebrew/lib/node_modules/npm/lib/commands/install.js:149:5)
49 verbose stack     at async module.exports (/opt/homebrew/lib/node_modules/npm/lib/cli-entry.js:61:5)
50 verbose cwd /Users/stragner/Developer/lumoscompany/hallo-contracts/hallo-contracts
51 verbose Darwin 22.5.0
52 verbose node v20.2.0
53 verbose npm  v9.8.1
54 error code ERESOLVE
55 error ERESOLVE unable to resolve dependency tree
56 error
57 error While resolving: hallo-contracts@0.0.1
57 error Found: ton-core@0.49.1
57 error node_modules/ton-core
57 error   dev ton-core@"^0.49.0" from the root project
57 error   peer ton-core@">=0.48.0" from @ton-community/blueprint@0.12.0
57 error   node_modules/@ton-community/blueprint
57 error     dev @ton-community/blueprint@"^0.12.0" from the root project
57 error
57 error Could not resolve dependency:
57 error peer ton-core@">=0.51.0" from ton@13.6.0
57 error node_modules/ton
57 error   dev ton@"^13.4.1" from the root project
57 error   peer ton@">=13.4.1" from @ton-community/blueprint@0.12.0
57 error   node_modules/@ton-community/blueprint
57 error     dev @ton-community/blueprint@"^0.12.0" from the root project
57 error
57 error Fix the upstream dependency conflict, or retry
57 error this command with --force or --legacy-peer-deps
57 error to accept an incorrect (and potentially broken) dependency resolution.
58 error
58 error
58 error For a full report see:
58 error /Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-eresolve-report.txt
59 verbose exit 1
60 timing npm Completed in 117ms
61 verbose unfinished npm timer reify 1692550861784
62 verbose unfinished npm timer reify:loadTrees 1692550861785
63 verbose unfinished npm timer idealTree:buildDeps 1692550861793
64 verbose unfinished npm timer idealTree:#root 1692550861793
65 verbose code 1
66 error A complete log of this run can be found in: /Users/stragner/.npm/_logs/2023-08-20T17_01_01_699Z-debug-0.log
```

**Comment by krigga** [comment]
Should be fixed now with create-ton@0.7.0, please reopen if the issue persists
